packages/regl-worldview/src/WorldviewContext.js

```diff
@@ -83,6 +85,10 @@ export class WorldviewContext {
   _drawCalls: Map<React.Component<any>, DrawInput> = new Map();
   _paintCalls: Map<PaintFn, PaintFn> = new Map();
   _hitmapObjectIdManager: HitmapObjectIdManager = new HitmapObjectIdManager();
+  _cachedReadHitmapCall: ?{
+    arguments: Array<any>,
+    result: Array<[MouseEventObject, Command<any>]>,
+  } = null;
```

Comment:
#nit I prefer avoiding `null` since by default it's already `undefined` and it's weird that JS has multiple concepts for essentially the same thing, so might be simpler to just always use `undefined`.